requirements.txt

```diff
@@ -1,5 +1,5 @@
-django>=1.11,<2.1
+django>=1.11,<2.2
 swapper>=1.1.0
 openwisp-utils>=0.2,<0.3
-djangorestframework>=3.7,<3.9
+djangorestframework>=3.7,<4.0
```

Comment:
change to `3.10`, we don't know if they're doing to release the `4.0`

Reply:
Hey... what should I change to "3.10"? I am not understading...

Reply:
You meant "4.10" right?

Reply:
`3.9` -> `3.10`!

Reply:
Great! I understood I think. I should apply this when I'm developing software too! Thanks for having introduced me to that.